en/hosting_guide/requirements.rst

```diff
@@ -1,14 +1,26 @@
 Requirements
 ============
 
-Webserver
----------
+Front servers
+-------------
+
+Production server
+~~~~~~~~~~~~~~~~~
 
 It's required to have a PHP webserver in order to run Open Orchestra.
 
 * PHP 5.4+
 * Apache 2 or NginX
 * Varnish 3 reverse proxy (optionnal)
+* Nodejs
```

Comment:
As it is a doc on production server, I think you should mention that Open Orchestra is "certified" on certain distrib as Debian.